__tests__/ApplicationTest.js

```diff
@@ -19,42 +19,63 @@ const mockRandoms = (numbers) => {
 };
 
 const getLogSpy = () => {
-  const logSpy = jest.spyOn(MissionUtils.Console, "print");
+  const logSpy = jest.spyOn(MissionUtils.Console, 'print');
   logSpy.mockClear();
   return logSpy;
 };
 
-describe("자동차 경주", () => {
-  test("기능 테스트", async () => {
+describe('자동차 경주', () => {
+  test('기능 테스트', async () => {
     // given
     const MOVING_FORWARD = 4;
     const STOP = 3;
-    const inputs = ["pobi,woni", "1"];
-    const logs = ["pobi : -", "woni : ", "최종 우승자 : pobi"];
+    const testCases = [
+      {
+        inputs: ['pobi,woni', '1'],
+        logs: ['pobi : ', 'woni : -', '최종 우승자 : woni'],
+        numbers: [STOP, MOVING_FORWARD],
+      },
+      {
+        inputs: ['oms, pobi, woni', '1'],
+        logs: ['oms : -', 'pobi : ', 'woni : ', '최종 우승자 : oms'],
+        numbers: [MOVING_FORWARD, STOP, STOP],
+      },
+    ];
     const logSpy = getLogSpy();
 
-    mockQuestions(inputs);
-    mockRandoms([MOVING_FORWARD, STOP]);
+    for (const { inputs, logs, numbers } of testCases) {
+      mockQuestions(inputs);
+      mockRandoms(numbers);
```

Comment:
for of도 좋지만 Object.entries를 사용하는 것이 더 나을 것 같아요~ [이런](https://github.com/ParkSB/javascript-style-guide?tab=readme-ov-file#iterators--nope) 컨벤션이 있더라구요!
```suggestion
    Object.entries(testCases).forEach(({ inputs, logs, numbers }) => {
      mockQuestions(inputs);
      mockRandoms(numbers);
```